service/src/main/java/bio/terra/workspace/service/workspace/flight/SyncSamGroupsStep.java

```diff
@@ -47,6 +47,9 @@ public StepResult doStep(FlightContext flightContext)
     workspaceRoleGroupMap.put(
         WsmIamRole.READER,
         samService.syncWorkspacePolicy(workspaceUuid, WsmIamRole.READER, userRequest));
+    workspaceRoleGroupMap.put(
```

Comment:
I don't think we need to sync the discoverer role. That will create Google groups for discoverers. We only need that when we are going to apply that group to a cloud resource ACL; like granting a READER actual read access to a GCS bucket.

I think we only ever enforce discoverer by checking with Sam - never the cloud.